src/librustc_typeck/astconv.rs

```diff
@@ -940,18 +934,16 @@ impl<'o, 'gcx: 'tcx, 'tcx> AstConv<'gcx, 'tcx>+'o {
                                   span: Span,
                                   ty: &hir::Ty,
                                   bounds: &[hir::TyParamBound])
-                                  -> Ty<'tcx>
-    {
-        /*!
-         * In a type like `Foo + Send`, we want to wait to collect the
-         * full set of bounds before we make the object type, because we
-         * need them to infer a region bound.  (For example, if we tried
-         * made a type from just `Foo`, then it wouldn't be enough to
-         * infer a 'static bound, and hence the user would get an error.)
-         * So this function is used when we're dealing with a sum type to
-         * convert the LHS. It only accepts a type that refers to a trait
-         * name, and reports an error otherwise.
-         */
+                                  -> Ty<'tcx> {
+        //! In a type like `Foo + Send`, we want to wait to collect the
+        //! full set of bounds before we make the object type, because we
+        //! need them to infer a region bound.  (For example, if we tried
+        //! made a type from just `Foo`, then it wouldn't be enough to
+        //! infer a 'static bound, and hence the user would get an error.)
+        //! So this function is used when we're dealing with a sum type to
+        //! convert the LHS. It only accepts a type that refers to a trait
+        //! name, and reports an error otherwise.
+        //!
```

Comment:
hmm, I guess the trailing blank `//!` is a feature, not a bug, of `rustfmt` (since IIUC it refuses to revise the layout of comments in terms of line breaks ... though I had thought @nrc had said something about it being willing to remove some blank lines in certain cases...)